src/System.Private.Xml/tests/System.Private.Xml.Tests.builds

```diff
@@ -3,65 +3,125 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="Readers\CharCheckingReader\System.Xml.RW.CharCheckingReader.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
When I try running these tests with `/p:FilterToTestTFM=netcoreapp1.0`, I hit https://github.com/dotnet/coreclr/issues/7607. Is there a way to work around that?